changelog/upgrade-notes.md

```diff
@@ -2,6 +2,13 @@
 
 ### v3.5.0
 
+#### `ol.tilegrid` changes
+
+* If you previously created an XYZ tilegrid, you could construct an `ol.tilegrid.XYZ`. This class has been replaced by a static `ol.tilegrid.createXYZ` function which takes the same options as the previous `ol.tilegrid.XYZ` constructor, but returns an `ol.tilegrid.TileGrid` instance. The internal tile coordinate scheme for XYZ tile grids has been changed. Previously, the `y` of tile coordinates was transformed to the coordinates used by sources by calculating `-y-1`. Now, it is transformed by calculating `height-y-1`, where height is the number of rows of the tile grid at the zoom level of the tile coordinate.
```

Comment:
No, I really meant tilegrid - if you previously created an XYZ source, then nothing changes. But maybe I should create a new bullet point before "The internal tile coordinate scheme for XYZ tile grids has been changed.", because that refers to XYZ sources.